### PR TITLE
feat : alarm rule ui 

### PR DESCRIPTION
**Pull request by wzh425**
1. feat : UI restoration of alarm rule related pages
2. chore : update masa nuget package
3. chore : project dependency adjustment

Please note that this time the pr contains https://github.com/masastack/MASA.Alert/pull/1

### CODE REVIEW

**Comment by doddgu** [comment]
@MayueCif PTAL, and merge it.

File: src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Base/BaseDaprCaller.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        if (webHostEnvironment.IsDevelopment())
+        {
+            var appIdSuffix = NetworkUtils.GetPhysicalAddress();
+
+            AppId = $"{AppId}-{appIdSuffix}";
+        }
+        else if (!webHostEnvironment.IsProduction())
+        {
+            AppId = $"{AppId}-{webHostEnvironment.EnvironmentName.ToLower()}";
+        }
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Base/BaseDaprCaller.cs:18]: else?

File: src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Extensions/HeaderRequestMessage.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿namespace Masa.Alert.ApiGateways.Caller.Extensions;
+
+public class HeaderRequestMessage : IDaprRequestMessage
```
Comment (MayueCif) [inline src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Extensions/HeaderRequestMessage.cs:3]: obsolete

File: src/Application/Masa.Alert.Application.Contracts/Enums/SearchTimeTypes.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Application.Contracts.Enums;
+
+public enum SearchTimeTypes
```
Comment (MayueCif) [inline src/Application/Masa.Alert.Application.Contracts/Enums/SearchTimeTypes.cs:6]: SearchTime？

File: src/Domain/Masa.Alert.Domain.Shared/AlarmRules/AlarmRuleCheckFrequency.cs [hunk 4]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Domain.Shared.AlarmRules;
+
+public enum AlarmRuleCheckFrequency
```
Comment (MayueCif) [inline src/Domain/Masa.Alert.Domain.Shared/AlarmRules/AlarmRuleCheckFrequency.cs:6]: AlarmCheckFrequencyTypes

File: src/Domain/Masa.Alert.Domain.Shared/Consts/AlertConsts.cs [hunk 5]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Domain.Shared.Consts;
+
+public static class AlertConsts
+{
+    public const string DbTablePrefix = null;
```
Comment (MayueCif) [inline src/Domain/Masa.Alert.Domain.Shared/Consts/AlertConsts.cs:8]: null？

File: src/Infrastructure/Masa.Alert.EntityFrameworkCore/EntityFrameworkCore/AlertDbContext.cs [hunk 6]
```diff
@@ -0,0 +1,15 @@
+﻿namespace Masa.Alert.EntityFrameworkCore.EntityFrameworkCore
```
Comment (MayueCif) [inline src/Infrastructure/Masa.Alert.EntityFrameworkCore/EntityFrameworkCore/AlertDbContext.cs:1]: file scope namespace

File: src/Domain/Masa.Alert.Domain.Shared/Consts/AlertConsts.cs (hunk 5 above)
Comment (wzh425) [reply src/Domain/Masa.Alert.Domain.Shared/Consts/AlertConsts.cs:8]: My table does not need a prefix at this time

File: src/Web/Masa.Alert.Web.Admin.Server/Pages/Login.cshtml.cs [hunk 7]
```diff
@@ -0,0 +1,31 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin.Server.Pages;
+
+public class LoginModel : PageModel
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin.Server/Pages/Login.cshtml.cs:6]: unnecessary？

File: src/Web/Masa.Alert.Web.Admin.Server/Pages/Logout.cshtml.cs [hunk 8]
```diff
@@ -0,0 +1,24 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin.Server.Pages;
+
+public class LogoutModel : PageModel
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin.Server/Pages/Logout.cshtml.cs:6]: unnecessary？

File: src/Web/Masa.Alert.Web.Admin/Data/Base/GlobalVariables.cs [hunk 9]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin.Data.Base
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/Data/Base/GlobalVariables.cs:4]: unnecessary？

File: src/Web/Masa.Alert.Web.Admin/Data/Shared/Favorite/FavoriteService.cs [hunk 10]
```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin.Data.Shared.Favorite
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/Data/Shared/Favorite/FavoriteService.cs:4]: unnecessary？

File: src/Web/Masa.Alert.Web.Admin/Global/Config/PageModes.cs [hunk 11]
```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin.Global.Config
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/Global/Config/PageModes.cs:4]: delete

File: src/Web/Masa.Alert.Web.Admin/ServiceCollectionExtensions.cs [hunk 12]
```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Alert.Web.Admin;
+
+public static class ServiceCollectionExtensions
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/ServiceCollectionExtensions.cs:6]: unnecessary

File: src/Web/Masa.Alert.Web.Admin/Shared/EmptyLayout.razor [hunk 13]
```diff
@@ -0,0 +1,7 @@
+﻿@inherits LayoutComponentBase
+
+<MApp>
+    <div style="height:100%;margin:0;padding:0;">
+        @Body
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/Shared/EmptyLayout.razor:5]: unnecessary

File: src/Web/Masa.Alert.Web.Admin/wwwroot/i18n/languageConfig.json [hunk 14]
```diff
@@ -0,0 +1,6 @@
+[
+  "zh-CN",
```
Comment (MayueCif) [inline src/Web/Masa.Alert.Web.Admin/wwwroot/i18n/languageConfig.json:2]: unnecessary file

**Review by MayueCif — COMMENTED** [review]
@wzh425

File: src/Application/Masa.Alert.Application.Contracts/Enums/SearchTimeTypes.cs (hunk 3 above)
Comment (wzh425) [reply src/Application/Masa.Alert.Application.Contracts/Enums/SearchTimeTypes.cs:6]: Used to query the data within the time range according to which time field in the application layer paging query

File: src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Base/BaseDaprCaller.cs (hunk 1 above)
Comment (wzh425) [reply src/ApiGateways/Caller/Masa.Alert.ApiGateways.Caller/Base/BaseDaprCaller.cs:18]: AppId

**Review by MayueCif — APPROVED** [review]
LGTM

**Pull request merged**